ob-pattern-1/templates/obkm/wso2ob-pattern-1-obkm-deployment.yaml

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wso2ob-pattern-1-obkm-deployment
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: {{ default 1 .Values.wso2.deployment.wso2obkm.replicas }} 
+  minReadySeconds: {{ default 30 .Values.wso2.deployment.wso2obkm.minReadySeconds }}
+  strategy:
+    rollingUpdate:
+      maxSurge: {{ default 1 .Values.wso2.deployment.wso2obkm.strategy.rollingUpdate.maxSurge }}
+      maxUnavailable: {{ default 0 .Values.wso2.deployment.wso2obkm.strategy.rollingUpdate.maxUnavailable }}
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      deployment: wso2ob-pattern-1-obkm
+  template:
+    metadata:
+      labels:
+        deployment: wso2ob-pattern-1-obkm
+    spec:
+      containers:
```

Comment:
@kdanW please check where we can integrate Kubernetes InitContainers and add them (e.g. for checking the availability of the DBMS server).

Please see the example [1].

[1]: [Kubernetes InitContainers in WSO2 IAM deployment](https://github.com/wso2/kubernetes-is/blob/v5.9.0.1/advanced/is-pattern-1/templates/is/identity-server-deployment.yaml#L44)

Reply:
Fixed with https://github.com/wso2/kubernetes-open-banking/pull/13/commits/a6c289bdb04eb294570ae486704103047497e89d